model/client4.go

```diff
@@ -5885,20 +5885,6 @@ func (c *Client4) DownloadExport(name string, wr io.Writer, offset int64) (int64
 	return n, BuildResponse(r)
 }
 
-func (c *Client4) GetThreadMentionsForUserPerChannel(userId, teamId string) (map[string]int64, *Response) {
```

Comment:
This is effectively our public API right? Can this not break external systems (plugins?) relying on this API call?

Reply:
this is an API specifically added for collapsed threads and is not used by anyone else but us @zefhemel